### PR TITLE
FPO-240-content

### PR DESCRIPTION
**Pull request by Lee365**
### Jira link
https://transformuk.atlassian.net/browse/FPO-240

### What?

I have added/removed/altered:

- [x] Added
- [x] Removed
- [x] Altered

### Why?

I am doing this because:

-

### AC

- Tested
- Shared with stakeholders
- Pair reviewed

### CODE REVIEW

File: views/checkVerification.njk [hunk 1]
```diff
@@ -1,16 +1,9 @@
 {% extends "layout.njk" %}
 {% block content %}
     {% include "back-button.njk" %}
-    <h1 class="govuk-heading-l">Check your answers before sending your application</h1>
+    <h1 class="govuk-heading-l">Check your answers</h1>
     <h2 class="govuk-heading-m">Application Details</h2>
     <table class="govuk-table">
```
Comment (ashward) [inline views/checkVerification.njk:6]: This should be a 'summary list' component, not a table. Ensure that we're following the pattern here: https://design-system.service.gov.uk/patterns/check-answers/ and using the nunjucks macros wherever possible.

File: views/checkVerification.njk [hunk 2]
```diff
@@ -1,16 +1,9 @@
 {% extends "layout.njk" %}
 {% block content %}
     {% include "back-button.njk" %}
```
Comment (ashward) [inline views/checkVerification.njk:3]: The back button should be in the `beforeContent` block so that it's outside of the `main` content.

Comment (Lee365) [reply views/checkVerification.njk:3]: To be covered in FPO-270

File: views/checkVerification.njk (hunk 1 above)
Comment (Lee365) [reply views/checkVerification.njk:6]: Changed to govukSummaryList

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**